Mathlib/Analysis/Fourier/FourierTransformDeriv.lean

```diff
@@ -214,6 +397,31 @@ lemma hasDerivAt_fourierIntegral
   rfl
 
 theorem deriv_fourierIntegral
-    {f : ℝ → E} (hf : Integrable f) (hf' : Integrable (fun x : ℝ ↦ x • f x)) (x : ℝ) :
-    deriv (𝓕 f) x = 𝓕 (fun x : ℝ ↦ (-2 * π * I * x) • f x) x :=
-  (hasDerivAt_fourierIntegral hf hf' x).deriv
+    {f : ℝ → E} (hf : Integrable f) (hf' : Integrable (fun x : ℝ ↦ x • f x)) :
+    deriv (𝓕 f) = 𝓕 (fun x : ℝ ↦ (-2 * π * I * x) • f x) := by
+  ext1 x
+  exact (hasDerivAt_fourierIntegral hf hf' x).deriv
+
+theorem iteratedDeriv_fourierIntegral {f : ℝ → E} {N : ℕ∞} {n : ℕ}
+    (hf : ∀ (n : ℕ), n ≤ N → Integrable (fun x ↦ x^n • f x)) (hn : n ≤ N) :
+    iteratedDeriv n (𝓕 f) = 𝓕 (fun x : ℝ ↦ (-2 * π * I * x) ^ n • f x) := by
+  ext1 x
+  have A (n : ℕ) (hn : n ≤ N) : Integrable (fun v ↦ ‖v‖^n * ‖f v‖) := by
+    convert (hf n hn).norm with x
+    simp [norm_smul]
+  have B : AEStronglyMeasurable f := by
+    convert (hf 0 (zero_le _)).1 with x
+    simp
+  have C : Integrable (fun v ↦ 𝐞 (-⟪v, x⟫_ℝ) • fourierPowSMulRight (innerSL ℝ) f v n) := by
+    simpa [-RCLike.inner_apply] using integrable_fourierPowSMulRight (A n hn) B
+  rw [iteratedDeriv, iteratedFDeriv_fourierIntegral A B hn, fourierIntegral_eq,
+    ContinuousMultilinearMap.integral_apply C, fourierIntegral_eq]
+  congr with y
+  suffices (-(2 * π * I)) ^ n • y ^ n • f y = (-(2 * π * I * y)) ^ n • f y by
+    simpa only [RCLike.inner_apply, conj_trivial, ContinuousMultilinearMap.smul_apply,
```

Comment:
This `simpa` is terminal and the expanded `simpa only` is rather long. Perhaps don't expand it, for readability? Unless its very slow that way.